src/btouch.cs

```diff
@@ -371,6 +371,8 @@ int Main3 (string [] args)
 					baselibdll = Path.Combine (GetSDKRoot (), "lib", "reference", "mobile", "Xamarin.Mac.dll");
 				else if (target_framework == TargetFramework.Xamarin_Mac_4_5_Full || target_framework == TargetFramework.Xamarin_Mac_4_5_System)
 					baselibdll = Path.Combine (GetSDKRoot (), "lib", "reference", "full", "Xamarin.Mac.dll");
+				else if (target_framework == TargetFramework.DotNet_5_0_macOS)
+					baselibdll = Path.Combine (GetSDKRoot (), "lib/mono/Xamarin.Mac/Xamarin.Mac.dll");
```

Comment:
Please, do the same as with the other lines that generate the baselibdll:
```suggestion
					baselibdll = Path.Combine (GetSDKRoot (), "lib", "mono", "Xamarin.Mac", "Xamarin.Mac.dll");
```

@rolfbjarne  do we have unit tests for this?

Reply:
@drasticactions you did a commit and you missed this one ;)

Reply:
I was following the pattern for what the other TFMs had, since those had hard paths too. I can convert them.

Reply:
Changed with https://github.com/xamarin/xamarin-macios/pull/13572/commits/d5548e267ea7cde5cacf6098c75136a79546c46e, I didn't change the other ones since they were already like that.

Reply:
I'll make sure the others are correct updated, thx!